tests/lib/snaps/store/test-snapd-refresh-control.v1/meta/hooks/gate-auto-refresh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+CONTROL_FILE="$SNAP_COMMON"/control
+
+snapctl refresh --pending > "$SNAP_COMMON"/debug.log
+
+if [ -f "$CONTROL_FILE" ]; then
+    COMMAND=$(cat "$CONTROL_FILE")
+    snapctl refresh "$COMMAND" >> "$SNAP_COMMON"/debug.log 2>&1
```

Comment:
perhaps you should use > instead of appending because then to compare in the test you make sure the debug.log file contains just the latest status

Reply:
Good point, thanks.

Reply:
Ah no, it actually needs to be this way. The file is first created above  with `> "$SNAP_COMMON"/debug.log`, here I need to append.